Makefile

```diff
@@ -162,6 +162,13 @@ ifeq (rocksdb,$(findstring rocksdb,$(COSMOS_BUILD_OPTIONS)))
   CGO_ENABLED=1
   BUILD_TAGS += rocksdb
   ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb
+	# inject brew dependencies on mac
+	# this assumes that if you are building on mac, you've installed
+	# rocksdb, snappy, lz4, and zstd via brew
+	ifeq ($(NATIVE_GO_OS),darwin)
```

Comment:
Maybe we should use a flag? This may break compiling against rocksdb build from source outside of brew?